src/AspNet.Security.OAuth.Vkontakte/VkontakteAuthenticationHandler.cs

```diff
@@ -61,9 +68,9 @@ public class VkontakteAuthenticationHandler : OAuthHandler<VkontakteAuthenticati
 
             var principal = new ClaimsPrincipal(identity);
             var context = new OAuthCreatingTicketContext(principal, properties, Context, Scheme, Options, Backchannel, tokens, payload);
-            context.RunClaimActions(payload);
+            context.RunClaimActions();
 
-            await Options.Events.CreatingTicket(context);
+            await Events.CreatingTicket(context);
```

Comment:
:+1: but as part of a different PR that fixes all the providers at once.

Thanks.